joblib/test/test_memory.py

```diff
@@ -819,3 +819,28 @@ def test_concurrency_safe_write(tmpdir, backend):
              if i % 3 != 2 else load_func for i in range(12)]
     Parallel(n_jobs=2, backend=backend)(
         delayed(func)(obj, filename) for func in funcs)
+
+
+def test_memory_recomputes_after_an_error_why_loading_results(tmpdir):
+    memory = Memory(tmpdir.strpath)
+
+    def func(arg):
+        return arg, time.time()
+
+    cached_func = memory.cache(func)
+    input_arg = 'arg'
+    arg, timestamp = cached_func(input_arg)
+
+    # Make sure the function is correctly cached
+    assert arg == input_arg
+
+    single_cache_item, = _get_cache_items(memory.cachedir)
```

Comment:
Yep it makes sure that the list of length 1 and then takes assign the single element value to the variable.
```py
l = [0]
x, = l  # x = 0
```